.gitignore

```diff
@@ -1,2 +1,2 @@
 # Apple folder misc info
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
No difference ? Right ?

Reply:
Just added a blank newline at the end of file, to remove the ugly '⛔' at the end of the line